components/select/select-option.tsx

```diff
@@ -108,13 +106,14 @@ const SelectOption: React.FC<React.PropsWithChildren<SelectOptionProps>> = ({
           height: 0;
           padding: 0;
           overflow: hidden;
-          border-top: 1px solid ${theme.palette.border};
+          border-top: ${theme.expressiveness.L1} ${theme.expressiveness.cLineStyle1}
+            ${theme.palette.border};
           margin: 0.5rem 0;
           width: 100%;
         }
 
         .label {
-          font-size: 0.875rem;
+          font-size: 1.1428rem;
```

Comment:
The font-size of the select label should be dynamic according to the size prop, same as the size prop to Select Component

Reply:
done